src/test/java/com/zufar/onlinestore/test/config/RestAssertion.java

```diff
@@ -0,0 +1,34 @@
+package com.zufar.onlinestore.test.config;
+
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import org.springframework.http.HttpStatus;
+
+import static io.restassured.module.jsv.JsonSchemaValidator.matchesJsonSchemaInClasspath;
+import static org.hamcrest.Matchers.*;
+
+public final class RestAssertion {
+
+    public static ValidatableResponse checkStatusCodeInResponse(Response response, int code, String schema) {
+        return response.then()
+                .statusCode(code)
+                .body(matchesJsonSchemaInClasspath(schema))
+                .time(lessThan(1500L));
+    }
+
+    public static void checkStatusCodeInResponse(Response response, int code) {
+        response.then()
+                .statusCode(code)
+                .body(emptyOrNullString())
+                .time(lessThan(1500L));
+    }
+
+    public static void baseNotFoundCheck(Response response, String schema) {
+        checkStatusCodeInResponse(response, HttpStatus.NOT_FOUND.value(), schema);
+    }
+
+    public static void baseOKHasItemsCheck(Response response, String path, String item, String schema) {
```

Comment:
I don't know how to name it better, but the current name is not clear enough

maybe `assertRestApiOkResponse` ?